source/content/guides/multisite/07-search-replace.md

```diff
@@ -0,0 +1,53 @@
+---
+title: WordPress Multisite
+subtitle: Search & Replace
+description: Additional handing required for running search and replace on WordPress Multisites.
+type: guide
+getfeedbackform: default
+contenttype: [guide]
+innav: [false]
+categories: [cms, troubleshooting]
+cms: [wordpress]
+audience: [development]
+product: [--]
+integration: [--]
+tags: [multisite]
+permalink: docs/guides/multisite/search-replace/
+editpath: multisite/07-search-replace.md
+---
+
+<Alert title="Early Access" type="info" icon="leaf">
+
+WordPress Multisite (WPMS) search and replace is available for [Early Access](/oss-support-levels#early-access) participants. Features for platform search and replace on WPMS sites are in active development. Pantheon's development team is rolling out new functionality often while this product is in Early Access. Visit [#wordpress in our community Slack](https://pantheon-community.slack.com/archives/CT8MC5Y0K) (you can sign up for the [Pantheon Slack channel here](https://slackin.pantheon.io/) if you don't already have an account) to learn how you can enroll in our Early Access program. Please review Pantheon's [Software Evaluation Licensing Terms](https://legal.pantheon.io/#contract-hkqlbwpxo) for more information about access to our software.
+
+</Alert>
+
+This section provides information on how to configure automatic platform search and replace of URLs for both subdirectory and subdomain WordPress Multisites.
+
+## Requirements
+A [WordPress Multisite](/guides/multisite)
+
+## Opting In
+To enable search replace between environments on a WPMS site, add 
```

Comment:
This could be `and` or `&`, but I think it needs one of them.
```suggestion
To enable search and replace between environments on a WPMS site, add 
```